Bitbucket-Pipelines/two-stage-pipeline-template/{{cookiecutter.outputDir}}/bitbucket-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+# Prerequisites: $AWS_ACCESS_KEY_ID and $AWS_SECRET_ACCESS_KEY setup in the Deployment variables https://support.atlassian.com/bitbucket-cloud/docs/set-up-and-monitor-deployments/.
+
```

Comment:
Can you please attach a screenshot of the pipeline this creates?

Reply:
Yep, sure.
The template tested as "Choice 2 - Custom Pipeline Template Location" during "sam pipeline init" workflow with test repository https://github.com/okyrdan/aws-sam-cli-template.git
It'll generate the next bitbucket-pipelines.yml file
![aws_sam_bitbucket_pipeliens](https://user-images.githubusercontent.com/13001581/132657342-091a43f8-2b49-46d6-ad70-68a4ebc4bd42.png)